app/indexers/essi/file_set_indexer.rb

```diff
@@ -6,8 +6,11 @@ class FileSetIndexer < Hyrax::FileSetIndexer
 
     def generate_solr_document
       super.tap do |solr_doc|
```

Comment:
For reviewers: The following calls to the parent resulted in four LDP load operations: once for each parent invocation, but _also_ for each safety check in the if clause.

Storing the parent once in a variable only loads it once but still gets access to the info for the properties.